core/src/main/java/org/opensearch/sql/expression/DSL.java

```diff
@@ -419,7 +419,12 @@ public static FunctionExpression time_to_sec(Expression... expressions) {
   }
 
   public static FunctionExpression timestamp(Expression... expressions) {
```

Comment:
Why leave this function signature?  There are only about 10 tests to update to remove this.

Reply:
All functions from this file are used in tests only.